app/controllers/peeps.rb

```diff
@@ -0,0 +1,20 @@
+class Chitter < Sinatra::Base
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps' do
+    user = User.first(id: env["rack.session"][:user_id])
+    @username = (user ? user[:username] : nil)
+    @peeps = Peep.all(:order => [:created_at.desc])
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)